README.md

```diff
@@ -37,6 +37,8 @@ Please check our [issues page](https://github.com/bellingcat/ukraine-timemap/iss
 * `XXXX_EXT` - points to the respective JSONs of the data, for events, sources, and associations
 * `MAPBOX_TOKEN` - used to load the custom styles
 * `DATE_FMT` and `TIME_FMT` - how to consume the events' date/time from the API
+* `store.app.language` - configures default language
```

Comment:
I'll rename `language` to `defaultLanguage` as it's easier to understand from just looking at the config